landavailability/tests/webclient/test_views.py

```diff
@@ -165,7 +165,7 @@ def test_search_view_postcode(self):
             'nearest_broadband_fast': None
         }]
 
-        api_url = '{0}/api/locations/'.format(
+        api_url = '{0}/api/location-search'.format(
             settings.LAND_AVAILABILITY_API_URL)
```

Comment:
Actually we already have such class https://github.com/alphagov/land-availability-api/blob/master/landavailability/api/permissions.py#L4